test/ShapeCrawler.Tests.Unit/TextFrameTests.cs

```diff
@@ -505,5 +505,134 @@ public void BottomMargin_getter_returns_bottom_margin_of_text_frame_in_centimete
             // Assert
             bottomMargin.Should().Be((decimal)expectedMargin);
         }
+
+        [Test]
+        public void SlideNotes_getter_returns_notes()
+        {
+            // Arrange
+            var pptxStream = StreamOf("056_slide-notes.pptx");
+            var pres = new Presentation(pptxStream);
+            var slide = pres.Slides[0];
+
+            // Act
+            var notes = slide.Notes;
+
+            // Assert
+            notes.Paragraphs.Should().HaveCount(4);
+            notes.Text.Should().Contain("NOTES LINE 1");
+        }
+
+        [Test]
+        public void SlideNotes_getter_enables_changing_notes()
+        {
+            // Arrange
+            var pptxStream = StreamOf("056_slide-notes.pptx");
+            var pres = new Presentation(pptxStream);
+            var slide = pres.Slides[0];
+            var notes = slide.Notes;
+            var expected = string.Join(Environment.NewLine, new[] { "0", "1", "2", "3" });
+
+            // Act
+            notes.Paragraphs[0].Text = "0";
+            notes.Paragraphs[1].Text = "1";
+            notes.Paragraphs[2].Text = "2";
+            notes.Paragraphs[3].Text = "3";
+
+            // Assert
+            notes.Paragraphs.Should().HaveCount(4);
+            notes.Text.Should().Be(expected);
+        }
+
+        [Test]
+        public void SlideNotes_getter_returns_null_if_no_notes()
+        {
+            // Arrange
+            var pptxStream = StreamOf("003.pptx");
+            var pres = new Presentation(pptxStream);
+            var slide = pres.Slides[0];
+
+            // Act
+            var notes = slide.Notes;
+
+            // Assert
+            notes.Should().BeNull();
+        }
+
+        [Test]
+        public void NewPresentation_has_no_notes()
+        {
+            // Arrange
+            var pres = new Presentation();
+            var slide = pres.Slides[0];
+
+            // Act
+            var notes = slide.Notes;
+
+            // Assert
+            notes.Should().BeNull();
+        }
+
+        [Test]
+        public void SlideAddNotes_adds_notes()
+        {
+            // Arrange
+            var pres = new Presentation();
+            var slide = pres.Slides[0];
+
+            // Act
+            slide.AddNotesIfEmpty();
+            var notes = slide.Notes;
+
+            // Assert
+            notes.Text.Should().Be(String.Empty);
+            pres.Validate();
+        }
+
+        [Test]
+        public void SlideAddNotes_can_change_notes()
+        {
+            // Arrange
+            var pres = new Presentation();
+            var slide = pres.Slides[0];
+            slide.AddNotesIfEmpty();
+            var notes = slide.Notes;
+            var expected = "SlideAddNotes_can_change_notes";
+
+            // Act
+            notes.Text = expected;
+
+            // Assert
+            notes.Text.Should().Be(expected);
+            pres.Validate();
+
+            // Just in case you want to check it!
```

Comment:
Please remove these comments.

I think we should document such guides in the out of the code comment, for example, in [CONTRIBUTING.md](https://github.com/ShapeCrawler/ShapeCrawler/blob/master/CONTRIBUTING.md).

You can also use the test method [SaveResult(pres)](https://github.com/ShapeCrawler/ShapeCrawler/blob/364bd96d407c43377105036b995d1d520b4a7451/test/ShapeCrawler.Tests.Unit.xUnit/Helpers/SCTest.cs#L63) which saves the test results in `\test\TestResults`